### ISSUE TITLE
[feature] trezorlib API readthedocs

### ISSUE THREAD

**Issue by rfjakob** [trezor/python-trezor#279]
Feature request: Ut would be nice to have the trezorlib API docs available via readthedocs.org or a similar service.

**Comment by matejcik** [comment]
indeed. it would even be nice to have api docs in the first place  ;)
